### PR TITLE
Lua cheat sheet: Added more operators

### PR DESCRIPTION
**Pull request by mayank7**
Added eight more operators to Lua cheat sheet.

IA Page: https://duck.co/ia/view/lua_cheat_sheet

### CODE REVIEW

**Comment by mayank7** [comment]
References:
http://www.lua.org/pil/3.1.html
http://www.lua.org/pil/3.2.html

File: share/goodie/cheat_sheets/json/lua.json [hunk 1]
```diff
@@ -47,6 +47,42 @@
...
+            {
+                "key": "<=",
+                "val": "Less than or equal"
+            },
+            {
+                "key": "+",
+                "val": "Addition"
+            },
+            {
+                "key": "- <binary>",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/lua.json]: I think the `<binary>` should be part of the description. I found it a little hard to understand what this was trying to indicate. In either case the operator or `-` so the key should be `-`

**Comment by mayank7** [comment]
@moollaza I put binary and unary in description

**Comment by moollaza** [comment]
Thanks! This LGTM :+1:

**Pull request merged**